File: trains-gdx-desktop/src/main/java/com/github/caay2000/trains/TrainsDesktop.java
package com.github.caay2000.trains;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;

public class TrainsDesktop {
	public static void main (String[] args) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();

		config.height = 1200;
		config.width = 1080;


		new LwjglApplication(new SimpleTest(), config);
	}
}

File: trains-gdx-desktop/src/main/kotlin/com/github/caay2000/trains/TrainsDesktop.kt
package com.github.caay2000.trains

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration

object TrainsDesktop {
    @JvmStatic
    fun main(args: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.height = 1200
        config.width = 1600
        LwjglApplication(SimpleTest(), config)
    }
}